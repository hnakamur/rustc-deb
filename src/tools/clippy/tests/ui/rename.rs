#![allow(stutter)]

#[warn(clippy::stutter)]
fn main() {}

#[warn(clippy::new_without_default_derive)]
struct Foo;

impl Foo {
    fn new() -> Self {
        Foo
    }
}
