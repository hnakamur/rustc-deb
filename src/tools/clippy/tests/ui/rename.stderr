error: unknown lint: `stutter`
  --> $DIR/rename.rs:1:10
   |
LL | #![allow(stutter)]
   |          ^^^^^^^
   |
   = note: `-D unknown-lints` implied by `-D warnings`

error: lint `clippy::stutter` has been renamed to `clippy::module_name_repetitions`
  --> $DIR/rename.rs:3:8
   |
LL | #[warn(clippy::stutter)]
   |        ^^^^^^^^^^^^^^^ help: use the new name: `clippy::module_name_repetitions`
   |
   = note: `-D renamed-and-removed-lints` implied by `-D warnings`

error: lint `clippy::new_without_default_derive` has been renamed to `clippy::new_without_default`
  --> $DIR/rename.rs:6:8
   |
LL | #[warn(clippy::new_without_default_derive)]
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: use the new name: `clippy::new_without_default`

error: aborting due to 3 previous errors

